fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/AbstractResourceIT.java

```diff
@@ -594,6 +594,15 @@ private static int countLinkHeader(final HttpResponse response, final String uri
         }).count();
     }
 
+    protected static String getOriginalResourceUri(final CloseableHttpResponse response) {
+        final List<Link> linkList = getLinkHeaders(response).stream()
+            .map(x -> Link.valueOf(x))
+            .filter(x -> x.getRel().equals("original"))
+            .collect(toList());
```

Comment:
since the assertions were removed, you could skip going to a list
```
return getLinkHeaders(response).stream()
            .map(x -> Link.valueOf(x))
            .filter(x -> x.getRel().equals("original"))
            .findFirst().get();
```